setup.py

```diff
@@ -48,7 +78,7 @@
     ],
     packages=find_packages(),
     include_package_data=True,
-    cmdclass=versioneer.get_cmdclass(),
+    cmdclass=__cmdclass,
```

Comment:
Interesting mechanism! Does this man the command is executed every time `repo2docker` is installed?
Could you also do it every time a new release is created?

@betatim @minrk Curious what you think about this.

Reply:
No, that command is only executed whenever you explicitly call `python setup.py generate_dataverse_file`. If you want it to be executed during `install` I'd need to override default `InstallCommand` and add something like `self.run_command("generate_dataverse_file")` to `InstallCommand.run()`.

BTW, I don't think it needs to be regenerated every release (definitely not every install). I added it so that it doesn't get lost.

Reply:
Thanks for the explanation!